### DISCUSSION TITLE
Allow for creation of 'block email/domain rule - similar to disable alias from email thread

### DISCUSSION THREAD

**Discussion by luigicalamar** [anonaddy/anonaddy#250] [Feature Requests]
Provide the capability to auto-create a rule for blocking a domain -or- email address directly from the email.  Add another option right below the 'Click here to deactivate this alias', such as:

_This email was sent to anonymous@anonaddy.com from spammer@spamdomain.com and has been forwarded by AnonAddy
Click here to deactivate this alias_
- 'Click here to block:  [spamdomain.com](spamdomain.com)' 
- 'Click here to block:  [spammer@spamdomain.com](url) address'

**Comment by willbrowningme** [comment]
Good idea, will look into this.

**Reply by luigicalamar** [reply]
Is this still on the 'books' as a feature enhancement?

**Reply by willbrowningme** [reply]
Yes, I have quite a lot on my to do list at the moment so cannot give a date when I expect this to be added I'm afraid.

**Comment by luigicalamar** [comment]
Np... Just checking.

Appreciate everything you do for anonaddy, it is a GREAT service

Luigi Calamar

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by simon-liebehenschel** [comment]
I am sorry what touching an old thread. 

> This email was sent to [anonymous@anonaddy.com](mailto:anonymous@anonaddy.com) from [spammer@spamdomain.com](mailto:spammer@spamdomain.com) and has been forwarded by AnonAddy
Click here to deactivate this alias

Why does this message is attached to some incoming emails?

UPDATE:

There are some discussions https://github.com/anonaddy/anonaddy/discussions?discussions_q=%22deactivate+this+alias%22 but it is not clear why some random emails have this banner attached.

**Reply by willbrowningme** [reply]
All forwarded messages have that banner information added. You can disable it or set it to appear at the bottom in your [account settings](https://app.anonaddy.com/settings) under "Update Email Banner Location" (requires a paid subscription).

**Reply by simon-liebehenschel** [reply]
Thank you. I have a paid account, so I will be able to change this.

> All forwarded messages have that banner information added.

Earlier I got emails from some people which are just a plain text without any banners attached. Maybe I am missing something, but if you will need more details for debugging let me know.
